backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/interest/InterestComputationCommand.java

```diff
@@ -71,6 +68,8 @@ public class InterestComputationCommand
 	 * we consider a fiscal year as having 12 months of 30 days
 	 */
 	public static final int TOTAL_DAYS_OF_FISCAL_YEAR = 360;
+	public static final long NO_INTEREST_DAYS = 0;
```

Comment:
```suggestion
	public static final long SUBTRACT_VALUE_NO_INTEREST_DAYS = 0;
```

As it is now, one can assume that it's used for both methods.